.github/workflows/macos.yml

```diff
@@ -0,0 +1,21 @@
+name: macos
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
I think we want to add "branch-*" to this so that the job will run when merged to branch-netatalk-3-1 for instance.